cinema/serializers.py

```diff
@@ -35,6 +45,10 @@ class MovieListSerializer(MovieSerializer):
         many=True, read_only=True, slug_field="full_name"
     )
 
+    class Meta:
+        model = Movie
+        fields = MovieSerializer.Meta.fields + ("genres", "actors")
```

Comment:
You inherit from MovieSerializer and it already has genres and actors fields in Meta